docs/root/start/quick-start.rst

```diff
@@ -136,77 +141,201 @@ The Envoy admin endpoint should also be available at http://localhost:9901
 
 .. literalinclude:: _include/envoy-demo.yaml
     :language: yaml
     :linenos:
     :lines: 1-3
     :emphasize-lines: 1
 
-Configuration: listeners
-************************
+.. _start_quick_start_static_listeners:
+
+Static configuration: ``listeners``
+***********************************
 
-The specification of the :ref:`listeners <envoy_v3_api_file_envoy/config/listener/v3/listener.proto>`.
+The example configures a :ref:`listener <envoy_v3_api_file_envoy/config/listener/v3/listener.proto>`
+on port ``10000``.
+
+All paths are matched and routed to the ``service_envoyproxy_io``
+:ref:`cluster <start_quick_start_static_clusters>`.
 
 .. literalinclude:: _include/envoy-demo.yaml
     :language: yaml
     :linenos:
-    :lines: 1-25
-    :emphasize-lines: 3-23
+    :lines: 1-29
+    :emphasize-lines: 3-27
 
+.. _start_quick_start_static_clusters:
 
-Configuration: clusters
-***********************
+Static configuration: ``clusters``
+**********************************
 
-The specification of the :ref:`clusters <envoy_v3_api_file_envoy/service/cluster/v3/cds.proto>`.
+The ``service_envoyproxy_io`` :ref:`cluster <envoy_v3_api_file_envoy/service/cluster/v3/cds.proto>`
+proxies over ``TLS`` to https://www.envoyproxy.io.
 
 .. literalinclude:: _include/envoy-demo.yaml
     :language: yaml
-    :lineno-start: 22
-    :lines: 22-47
-    :emphasize-lines: 4-24
+    :lineno-start: 27
+    :lines: 27-50
+    :emphasize-lines: 3-22
+
+.. _start_quick_start_static_admin:
 
-Configuration: admin
-********************
+Static configuration: ``admin``
+*******************************
 
-The :ref:`admin message <envoy_v3_api_msg_config.bootstrap.v3.Admin>` is required to configure
+The :ref:`admin message <envoy_v3_api_msg_config.bootstrap.v3.Admin>` is required to enable and configure
 the administration server.
 
 The ``address`` key specifies the listening :ref:`address <envoy_v3_api_file_envoy/config/core/v3/address.proto>`
 which in the demo configuration is ``0.0.0.0:9901``.
 
 .. literalinclude:: _include/envoy-demo.yaml
     :language: yaml
-    :lineno-start: 45
-    :lines: 45-50
-    :emphasize-lines: 3-6
+    :lineno-start: 48
+    :lines: 48-55
+    :emphasize-lines: 3-8
 
 .. warning::
 
    You may wish to restrict the network address the admin server listens to in your own deployment.
 
+.. _start_quick_start_dynamic:
 
 Dynamic configuration
 ---------------------
 
-See the :ref:`configuration overview <config_overview>` for further information on configuring Envoy with static and dynamic configuration.
+Setting up Envoy with dynamic configuration is slightly more complex as you must also set up a control plane
+to provide Envoy with its configuration.
+
+There are a number of control planes compatible with Envoy's API such as `Gloo <https://docs.solo.io/gloo/latest/>`_
+or `Istiod <https://istio.io/latest/docs/ops/deployment/architecture/#istiod>`_.
```

Comment:
I would probably just link directly to the Istio project and not istiod?

Reply:
ok - i found that link searching for the best explanation as to how you can use istiod as a control plane independent of istio - previously i could have linked to pilot - but the links no longer work and it seems like the docs just refer to istiod (in latest)

Reply:
removing dynamic info from this PR - ill deal with this in subsequent..